x-pack/test/functional/services/machine_learning/job_wizard_common.ts

```diff
@@ -166,6 +181,21 @@ export function MachineLearningJobWizardCommonProvider({ getService }: FtrProvid
       expect(await this.getSelectedJobGroups()).to.contain(jobGroup);
     },
 
+    async getSelectedCalendars(): Promise<string[]> {
+      return await comboBox.getComboBoxSelectedOptions(
```

Comment:
This should call `ensureAdditionalSettingsSectionOpen` first just to be less dependent on the state of the UI.

Reply:
Added in https://github.com/elastic/kibana/pull/52269/commits/94da14ea402f329134c8cb021231ca588b7f5890